front/src/Apps/Dashboard/DashboardRoutes.tsx

```diff
@@ -192,6 +179,20 @@ function DashboardRoutes() {
                 companies={filter(Exports.fragments.company, companies)}
               />
             </Route>
+
+            <Route path={routes.dashboardv2.bsds.drafts}>
```

Comment:
Nit: je crois que tu peux faire:
```
<Route path={[routes.dashboardv2.bsds.drafts, routes.dashboardv2.bsds.act, ...]}>
{dashboardPageComponent}
</Route>
```

Ce qui augmente la lisibilité je trouve puisqu'on comprend tout de suite que toutes redirigent vers le même composant

Reply:
of course! bien vu @GaelFerrand